app/src/main/java/io/github/droidkaigi/confsched2017/view/activity/SplashActivity.java

```diff
@@ -73,7 +71,8 @@ private void loadSessionsForCache() {
                     if (isFinishing()) return;
                     startActivity(MainActivity.createIntent(SplashActivity.this));
                 })
-                .subscribe();
+                .subscribe(objectObservable -> Timber.tag(TAG).d("Succeeded in loading sessions."),
```

Comment:
Can you change valuable to `observable` from `objectObservable`?
In fact, lambda variable should be more short name like `o`, `t`.